src/js/config.js

```diff
@@ -6,7 +6,17 @@ const datadogConfig = {};
 const appConfig = {};
 const versions = {};
 
-function fetchConfig(success) {
+// Temporary solution for getting env ws url
```

Comment:
this is likely temporary as we'll either get some sort of stable url like `ws.demo.roundingwell.com`